src/Adapter/Order/QueryHandler/GetOrderForViewingHandler.php

```diff
@@ -138,7 +144,8 @@ public function __construct(
         Locale $locale,
         Context $context,
         CustomerDataProvider $customerDataProvider,
-        GetOrderProductsForViewingHandlerInterface $getOrderProductsForViewingHandler
+        GetOrderProductsForViewingHandlerInterface $getOrderProductsForViewingHandler,
+        Configuration $configuration
```

Comment:
I'm personally not a fan of including the whole configuration as it makes it harder to test. But it's not worse than using the static version...

Reply:
Would you rather provide an array of specific configuration keys and its values? Something like we used to do in `Configuration::getMultiple`? I'm just curious.

I can imagine some scenarios where tests could fail if not executed in proper order but from what I know we're resetting configuration before tests so... it shouldn't be an issue right?